pkg/META

```diff
@@ -1,4 +1,4 @@
-version = "%%VERSION_NUM%%"
+version = "0.19.1"
```

Comment:
this shouldn't be done in git.. if you use `topkg distrib` and `topkg opam pkg`, the watermarks are replaced in the resulting tarball... no need to change in the repository